### PR TITLE
feat(alephalpha): Add AlephAlpha instrumentation

### PR DESCRIPTION
**Pull request by Benedikt-Wolf**
Added support for Aleph Alpha's python client. Currently only for synchronous completions.

- [x] I have added tests that cover my changes.
- [ ] If adding a new instrumentation or changing an existing one, I've added screenshots from some observability platform showing the change.
- [x] PR name follows conventional commits format: `feat(instrumentation): ...` or `fix(instrumentation): ...`.
- [x] (If applicable) I have updated the documentation accordingly.

### CODE REVIEW

File: packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py [hunk 1]
```diff
@@ -865,3 +874,21 @@ def init_weaviate_instrumentor():
...
+            from opentelemetry.instrumentation.alephalpha import AlephAlphaInstrumentor
+
+            instrumentor = AlephAlphaInstrumentor(
+                exception_logger=lambda e: Telemetry().log_exception(e),
+            )
+            if not instrumentor.is_instrumented_by_opentelemetry:
+                instrumentor.instrument()
+        return True
+    except Exception as e:
+        logging.error(f"Error initializing TogetherAI instrumentor: {e}")
```
Comment (nirga) [inline packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py:892]: AlephAlpha

File: packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py [hunk 2]
```diff
@@ -865,3 +874,21 @@ def init_weaviate_instrumentor():
         logging.warning(f"Error initializing Weaviate instrumentor: {e}")
         Telemetry().log_exception(e)
         return False
+
+
+def init_alephalpha_instrumentor():
+    try:
+        if importlib.util.find_spec("alephalpha") is not None:
```
Comment (nirga) [inline packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py:881]: Isn't the name of the package `alephalpha-client`?

**Review by nirga — COMMENTED** [review]
Great work @Benedikt-Wolf! Left a couple of small comments

File: packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py (hunk 2 above)
Comment (Benedikt-Wolf) [reply packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py]: Oversaw that, my bad. I changed it to "aleph_alpha_client".

**Review by nirga — APPROVED** [review]

**Pull request merged**